src/FluxRM.jl

```diff
@@ -11,4 +13,56 @@ function version()
     Base.VersionNumber(major[], minor[], patch[])
 end
 
+mutable struct Broker
```

Comment:
@garlick @SteVwonder 

I called this `Broker` instead of `Flux` since I wanted to prevent a name clash with `Flux.jl`,
does this seem like an appropriate name?

Reply:
Thoughts on `FluxHandle`?  Or `Handle`?  I think we usually use `broker` to refer to the actual process and `handle` to refer to the connection into the broker.  (Speaking under correction from @garlick and others of course).

Reply:
Hm `Handle` is very broad since there will be many handles, like to futures.

Reply:
> since there will be many handles, like to futures.

Can those be called `Future`?  That's at least [what we've done for python](https://github.com/flux-framework/flux-core/blob/master/src/bindings/python/flux/future.py#L55).  Not sure how well that maps to Julia and its idioms.

Reply:
Something I might be missing is how Julia does namespacing.  If you do `using FluxRM`, does that bring all exported names into the current scope?  For example:

```julia
using FluxRM

flux = Handle()
```

Or would it need to be `FluxRM.Handle()`?  I guess what I'm really asking is, do you have to worry about collisions with exported names from other packages?

Reply:
> Can those be called Future? That's at least what we've done for python.

Yeah that will be fine.

https://docs.julialang.org/en/v1/manual/modules/#Summary-of-module-usage

In short `using FluxRM` will bring all `export`ed names into the namespace.

```
using FluxRM
using Flux
```

will be fine, but

```
using FluxRM

flux = Flux()

using Flux
```

will error with:

```
WARNING: using Flux.Flux in module Main conflicts with an existing identifier.
ERROR: importing Flux into Main conflicts with an existing identifier
```

might be fine though, since one can always use `FluxRM.Flux` or do `import FluxRM` instead of `using FluxRM`.